src/go/nerdctl-stub/nerdctl_commands_generated.go

```diff
@@ -158,12 +157,41 @@ var commands = map[string]commandDefinition{
 	"builder": {
 		commandPath: "builder",
 		subcommands: map[string]struct{}{
+			"build": {},
 			"debug": {},
 			"prune": {},
 		},
 		options: map[string]argHandler{},
 	},
 
+	"builder build": {
+		commandPath: "builder build",
+		subcommands: map[string]struct{}{},
+		options: map[string]argHandler{
+			"--build-arg":     ignoredArgHandler,
+			"--buildkit-host": ignoredArgHandler,
+			"--cache-from":    ignoredArgHandler,
+			"--cache-to":      ignoredArgHandler,
+			"--file":          ignoredArgHandler,
+			"--iidfile":       ignoredArgHandler,
```

Comment:
Same  here… and probably some more.

Reply:
We'd need to write more `*ArgHandler` functions, basically. I'll deal with it.

Reply:
In this PR or a future one?

Reply:
> In this PR or a future one?

Mark has already written them in this PR; that's what all the code changes are about. The rddepman commit just bumps the version number for wsl-distro. Everything else is about dealing with new options that take file/directory name arguments.